doc/howto/cross_compiling/cross_compiling_for_raspberry.md

```diff
@@ -0,0 +1,61 @@
+# Build PaddlePaddle for Raspberry Pi
+You may use any of the following two approaches to build the inference library of PaddlePaddle for Raspberry Pi:
+
+1. Build it: Log in to a Raspberry Pi via SSH and build the library. The required development tools and third-party dependencies are listed in [`/Dockerfile`](https://github.com/PaddlePaddle/Paddle/blob/develop/Dockerfile). 
+
+1. Cross-compile: This article explains how to cross-compile PaddlePaddle for Raspberry Pi on a Linux/x64 computer.
+
+## The Cross-Compiling Toolchain
+
+Step 1. Clone the following Github repo
+
+```bash
+git clone https://github.com/raspberrypi/tools.git
+```
+
+Step 2. Use the pre-built cross-compiler found in `./tools/tree/master/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64`.  To run it on a Linux computer, glibc version >= 2.14 is needed.
+
+## CMake Arguments
+
+CMake supports [cross-compiling](https://cmake.org/cmake/help/v3.0/manual/cmake-toolchains.7.html#cross-compiling).  All CMake configuraition arguments required for the cross-compilation for Raspberry Pi can be found in [`cmake/cross_compiling/raspberry_pi.cmake`](https://github.com/PaddlePaddle/Paddle/blob/develop/cmake/cross_compiling/raspberry_pi.cmake).
+
+Some important arguments that need to be set:
+
+- `CMAKE_SYSTEM_NAME`: The target platform.  Must be `RPi`.
+
+- `RPI_TOOLCHAIN`: The absolute path of the cross-compiling toolchain.
+
+- `RPI_ARM_NEON`: Use ARM NEON Intrinsics. Must be and default to `ON`.
```

Comment:
Rephrase: `RPI_ARM_NEON`: Use ARM NEON Intrinsics. This is a required argument and set default to `ON`.

Reply:
Done